### PR TITLE
logstore: Return structured log lines for printing to stdout

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan, @maiamcc,

Please review the following commits I made in branch nicks/continuingstring:

902a694256e1f2cfc52d3e6731360bdf7e767390 (2020-01-28 13:15:50 -0500)
logstore: Return structured log lines for printing to stdout

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: pkg/model/logstore/logstore.go [hunk 1]
```diff
@@ -532,10 +568,41 @@ func (s *LogStore) startAndLastIndices(spans map[SpanID]*Span) (startIndex, last
...
 }
 
-func (s *LogStore) logHelper(spansToLog map[SpanID]*Span, showManifestPrefix bool) string {
+type logOptions struct {
+	spans                       map[SpanID]*Span
+	showManifestPrefix          bool
+	skipFirstLineManifestPrefix bool
+}
+
+func (s *LogStore) logHelper(options logOptions) string {
```
Comment (jazzdan) [inline pkg/model/logstore/logstore.go]: nit: This is pre-existing but I don't think `logHelper` is a good name for this function. `logOptionsToString` maybe?

File: pkg/model/logstore/logstore.go [hunk 2]
```diff
@@ -532,10 +568,41 @@ func (s *LogStore) startAndLastIndices(spans map[SpanID]*Span) (startIndex, last
...
+	return linesToString(s.logHelperLines(options))
+}
+
+// Returns a sequence of lines, including trailing newlines.
+func (s *LogStore) logHelperLines(options logOptions) []LogLine {
```
Comment (jazzdan) [inline pkg/model/logstore/logstore.go]: nit: Same thing here. `logOptionsToLine`?

File: pkg/model/logstore/logstore.go (hunk 1 above)
Comment (nicks) [reply pkg/model/logstore/logstore.go]: i changed it to `toLogString`... didn't want to imply that we were converting the logOptions struct itself into a string

File: pkg/model/logstore/logstore.go (hunk 2 above)
Comment (nicks) [reply pkg/model/logstore/logstore.go]: done

**Pull request merged**